x-pack/plugins/security/server/routes/views/access_agreement.ts

```diff
@@ -57,8 +57,7 @@ export function defineAccessAgreementRoutes({
 
         return response.ok({ body: { accessAgreement } });
       } catch (err) {
-        logger.error(err);
-        return response.internalError();
+        throw err;
       }
```

Comment:
__nit:__ same nit here, I believe we don't need `try\catch` just to re-throw error here, would you mind ditching it?

Reply:
Nice catch! Changed! Thank you! Also the applied the suggested changes in https://github.com/elastic/kibana/pull/85541#discussion_r572807243